lib/cloak/migrator.ex

```diff
@@ -18,6 +26,19 @@ defmodule Cloak.Migrator do
     |> Flow.run()
   end
 
+  defp migrate_schema(repo, schema, [:binary_id]) do
+    fields = cloak_fields(schema)
+
+    repo.all(from(s in schema, select: s.id))
```

Comment:
Loading all record IDs might not work well on a large table with millions of rows. We could use `Repo.stream`, but that would require a long-running transaction, which would also cause problems on a large table. 

Perhaps we could alter the strategy to look at the `:inserted_at` timestamp instead of the ID?

Reply:
I agree about performance on large tables, I'm scratching my own itch on this since the possibility of low performance is better than no option IMO (and I only have a few hundred rows at this point). `:inserted_at` isn't guaranteed to be unique, so would it work properly for handling this? I started with a long-running transaction.

I am not as familiar with Postgrex or Ecto database cursors, however would a cursor be a good option? Or perhaps A multi-stage migration that extends the target schema with a temporary monotonic integer ID field, then executes the cloak migration, then drops the temporary field?

Reply:
Yeah, now that I think about it more, `:inserted_at` wouldn't work. I'll look into cursors.

Reply:
I'm making good progress on a cursor-based solution, just so you know. Hopefully I'll be able to get something up this week. In the meantime, maybe you can use your fork to do your migration?